CHANGES.md

```diff
@@ -1,8 +1,491 @@
-====================
 Release Changes List
 ====================
 
+
+version 1.15.0
 ==============
+
+Eighteenth public release of Chapel, April 6, 2017
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* performance optimization highlights:
+  - significantly improved the performance of many key idioms and benchmarks
+  - dramatically improved task creation speed
+  - reduced overheads related to array slicing, rank-change, and reindexing
+  - significantly improved the remote value forwarding optimization
+  - optimized the performance of latency-bound applications with CHPL_COMM=ugni
+  - improved qthreads tasking such that it outperforms 'muxed' on studied codes
+* standard library/package highlights:
+  - added a new 'DateTime' module for operating on dates and times
+  - added BLAS level 1 and 2 routines to the previous support for level 3
+  - added an early-draft 'LinearAlgebra' module
+  - added a new 'Futures' module supporting library-based futures
+  - added initial support for some class-owner patterns, 'Owned' and 'Shared'
+  - added support for choosing between implementations for the 'FFTW' module
```

Comment:
This is also supported in BLAS with the config param: `isBLAS_MKL`, but not sure if it's noteworthy.